modules/localstack/examples_test.go

```diff
@@ -113,3 +116,83 @@ func ExampleRunContainer_legacyMode() {
 	// Output:
 	// version=localstack/localstack:0.10.0. Testcontainers for Go does not support running LocalStack in legacy mode. Please use a version >= 0.11.0
 }
+
+func ExampleRunContainer_UsingLambdas() {
+	ctx := context.Background()
+
+	flagsFn := func() string {
+		labels := testcontainers.GenericLabels()
+
+		flags := ""
+		for k, v := range labels {
+			flags = fmt.Sprintf("%s -l %s=%s", flags, k, v)
+		}
+
+		return flags
+	}
```

Comment:
See https://github.com/localstack/localstack/issues/8616#issuecomment-1741278016

I noticed this does not work with localstack v1.4.0, that's why I moved the example to v2.3.0